docs/guidelines/custom-elements-warp/index.md

```diff
@@ -0,0 +1,242 @@
+# Building Custom Elements with Warp
+
```

Comment:
This is probably self-explanatory, but should we emphasize that this is specifically for Web Components and that it is not related to React or Vue? :)

Reply:
Yeah custom elements === web components